virtual-classroom/src/main/java/ro/utcn/sd/mid/assign1/virtualclassroom/entity/Answer.java

```diff
@@ -0,0 +1,48 @@
+package ro.utcn.sd.mid.assign1.virtualclassroom.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity(name = "answers")
+@Getter
+@Setter
+@AllArgsConstructor @NoArgsConstructor
+public class Answer implements IDEntity{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    private Integer questionId;
+    private Integer userId;
+    private String text;
+    private Timestamp creationDate;
+    private Integer score;
+
+    public Answer(Integer questionID, Integer userID, String text) {
+        this.questionId = questionID;
+        this.userId = userID;
+        this.text = text;
+        creationDate = new Timestamp(System.currentTimeMillis());
+        score = 0;
+    }
+
+    @Override
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    @Override
+    public Integer getId() {
+        return id;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
You can also generate toString methods with the Lombok `@ToString` annotation.

Reply:
I wanted to display the score for each Answer in a special format, i.e. [score] Answer(id=1, ...)

Reply:
Ok. Just wanted to make sure that you know about the annotation :)